unix_setup.sh

```diff
@@ -55,15 +55,15 @@ else
     exit 1
 fi
 
-ext_jar_filepath=$PWD/autopsy/modules/ext/sleuthkit-postgresql-$TSK_VERSION.jar;
+ext_jar_filepath="$PWD/autopsy/modules/ext/sleuthkit-postgresql-$TSK_VERSION.jar";
```

Comment:
Not a big deal but you don't technically need to add the quotes here. The ext_jar_filepath variable gets set correctly with or without the quotes. It only needs to be wrapped in quotes when it is used in commands (rm and cp) below.

Reply:
Thanks, for reviewing and for the comment.

Reply:
You're welcome. Thanks for the submission :-)